dagger/pipeline/ios/dynamo_io.py

```diff
@@ -0,0 +1,47 @@
+from dagger.pipeline.io import IO
```

Comment:
shall we add unit tests for the new ios? 
we had unit tests for all other ios

Reply:
Added tests. Also found a bug with them :) Thanks!